### PR TITLE
Fixing the button size for mobile.

### PR DESCRIPTION
**Pull request by LabibAfsarAhmed**
- Fix the button size so that it changes according to the mobile device.

### CODE REVIEW

File: src/components/FeaturedProject.css [hunk 1]
```diff
@@ -66,14 +66,15 @@
...
 }
 
-.project__button:hover{
-  box-shadow: 2px 4px #4dd6a8;
-  /*(--turquoise-green);*/
-  transform: translateY(-0.2em);
-}
+@media (min-width: 400px){
+  .project__button:hover{
+    box-shadow: 2px 4px #4dd6a8;
```
Comment (Niyousha99) [inline src/components/FeaturedProject.css:71]: Could you replace the box-shadow hex color with the global variable `--primary-green` provided in App.css?

**Review by Niyousha99 — CHANGES_REQUESTED** [review]
I'm not feeling the "->" in the project button text. How about we remove it?

**Review by Niyousha99 — APPROVED** [review]
LGTM!

**Review by ss1106 — APPROVED** [review]
LGTM

**Pull request merged**